bigg_com.clj

```diff
@@ -0,0 +1,77 @@
+(config
+(password-field
+:name "bearerToken" 
+:label "<Provide label here>"))
+
+(default-source
+(header-params "Accept" "application/json")
+    (paging/page-number
+        :page-number-query-param-initial-value 0
+        :page-number-query-param-name "page"
+        :limit 10 
+        :limit-query-param-name "limit" 
+    )
+                    
+(auth/http-bearer)
+(http/get :base-url "https://developer.bigcommerce.com/docs/rest-management")
```

Comment:
Please verfiy the base once ( we will add the base URL for the endpoint which we will not  the doc URL)